packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -43,7 +43,11 @@ test('simpleExchange with valid offers', async t => {
     const {
       extent: [{ instanceHandle }],
     } = await inviteIssuer.getAmountOf(simonInvite);
-    const { publicAPI } = zoe.getInstance(instanceHandle);
+    const {
```

Comment:
I think we shouldn't combine these two calls if we intend for the code for Alice and Bob to be separate (even though it's all in the same file).

Reply:
done.